target_api/sinks.py

```diff
@@ -85,3 +85,19 @@ def make_batch_request(self, records: List[dict]):
             self.logger.warning(f"Unable to get response's id: {e}")
 
         return id, response.ok, dict()
+
+    def process_batch(self, context: dict) -> None:
```

Comment:
@keyn4 Why did we need this override? What's the change?

Reply:
the only change is adding the try except, and it's because without that it doesn't write the `target_state` because it fails before handling the response, and for big payloads I think the full curl won't show in the UI so it's better to have it in the `target_state` file